backend/akt/src/main/java/fi/oph/akt/scheduler/ExpiringAuthorisationsEmailCreator.java

```diff
@@ -33,18 +29,24 @@ public class ExpiringAuthorisationsEmailCreator {
   @Resource
   private final ClerkEmailService clerkEmailService;
 
-  @Scheduled(initialDelayString = INITIAL_DELAY, fixedDelayString = FIXED_DELAY)
+  @Scheduled(cron = "0 0 3 * * *")
   @SchedulerLock(name = "checkExpiringAuthorisations", lockAtLeastFor = LOCK_AT_LEAST, lockAtMostFor = LOCK_AT_MOST)
   public void checkExpiringAuthorisations() {
     SchedulingUtil.runWithScheduledUser(() -> {
-      LOG.debug("checkExpiringAuthorisations");
+      LOG.info("checkExpiringAuthorisations");
       final LocalDate expiryBetweenStart = LocalDate.now();
       final LocalDate expiryBetweenEnd = expiryBetweenStart.plusMonths(3);
       final LocalDateTime previousReminderSentBefore = expiryBetweenStart.minusMonths(4).atStartOfDay();
 
       authorisationRepository
         .findExpiringAuthorisations(expiryBetweenStart, expiryBetweenEnd, previousReminderSentBefore)
-        .forEach(clerkEmailService::createAuthorisationExpiryEmail);
+        .forEach(authorisationId -> {
+          try {
```

Comment:
Lisäsin tänne nyt myös try-catchin. Suoritetaan nyt forEachin sisällä joten vähän niin ja näin onko fiksu. Voidaan poistaa tai toteuttaa jollain muulla tavoin jos tällainen virhelokiprintti halutaan säilyttää